packages/core/src/routes/interaction/actions/submit-interaction.ts

```diff
@@ -212,6 +213,13 @@ export default async function submitInteraction(
 
     await assignInteractionResults(ctx, provider, { login: { accountId: id } });
 
+    const details = await trySafe(provider.interactionDetails(ctx.req, ctx.res));
+    ctx.addTriggerInteractionHookEvent(InteractionEvent.Register, {
+      sessionId: details?.jti,
+      userId: details?.result?.login?.accountId,
+      applicationId: details?.params.client_id,
```

Comment:
The interaction details info can be retrieved directly from ctx.details? Also accountId?

Reply:
Yeah, updated.